.gitignore

```diff
@@ -1 +1,5 @@
 build/
+.project
+/.gitignore
```

Comment:
we should also remove /.gitignore . Not sure of a case when this should be ignored.